ibft/storage/stores.go

```diff
@@ -2,24 +2,24 @@ package storage
 
 import (
 	"github.com/cornelk/hashmap"
-	spectypes "github.com/ssvlabs/ssv-spec/types"
+	"github.com/ssvlabs/ssv/exporter/exporter_message"
 
 	qbftstorage "github.com/ssvlabs/ssv/protocol/v2/qbft/storage"
 	"github.com/ssvlabs/ssv/storage/basedb"
 )
 
 // QBFTStores wraps sync map with cast functions to qbft store
 type QBFTStores struct {
-	m *hashmap.Map[spectypes.RunnerRole, qbftstorage.QBFTStore]
+	m *hashmap.Map[exporter_message.RunnerRole, qbftstorage.QBFTStore]
```

Comment:
maybe `exporter_message.RunnerRole` types and functions and better fitting to this package since they are used mostly here? or when dealing with the storage here.